### PR TITLE
AI Law Update

### LINKED ISSUE #1301: Fix AI's lawset

### PR DESCRIPTION
**Pull request by NanakoAC**
Asimov is really not working out in practise, it was always meant to be temporary.

This PR reworks the nanotrasen lawset and sets it back to the default. Naturally all mentions of NT are eris now.
Although these laws aren't explicitly hierarchical, the obey law is nevertheless moved to the top to pointedly illuminate the servile status of AIs. They shouldn't be overrulling command staff

fixes #1301 
also fixes a leftover global debug message in mind.dm

### CODE REVIEW

File: code/datums/ai_law_sets.dm [hunk 1]
```diff
@@ -11,22 +11,22 @@
 	..()
 
 /******************** Nanotrasen/Malf ********************/
-/datum/ai_laws/nanotrasen
+/datum/ai_laws/eris
 	name = "NT Default"
```
Comment (Kurgis) [inline code/datums/ai_law_sets.dm]: Shouldn't this be renamed to "Eris Default"?

File: code/game/objects/items/weapons/AI_modules.dm [hunk 2]
```diff
@@ -284,7 +284,7 @@ AI MODULES
 	name = "'NT Default' Core AI Module"
```
Comment (Kurgis) [inline code/game/objects/items/weapons/AI_modules.dm]: Shouldn't this be renamed to "Eris Default"?

File: code/datums/ai_law_sets.dm (hunk 1 above)
Comment (NanakoAC) [reply code/datums/ai_law_sets.dm]: GOOD POINT
will fix

File: code/game/objects/items/weapons/AI_modules.dm (hunk 2 above)
Comment (LethalGhost) [reply code/game/objects/items/weapons/AI_modules.dm]: May be `/obj/item/weapon/aiModule` should rebuild its name in `Initialize`

**Pull request merged**